### ISSUE TITLE
Dynamic compiled cloudctl v0.12.1 not usable with Ubuntu 20.04

### ISSUE THREAD

**Issue by gc-jro** [fi-ts/cloudctl#252]
Hi,

we tried the new cloudctl v0.12.1 and can't use it with Ubuntu 20.04.
The prior versions were all statically linked. The new version is dynamically linked.
Unfortunately, Ubuntu 20.04 doesn't use the required GLIBC version.

```bash
jens@[t3-nbg]:~/zf/zfag-tde-prototype$ podman run -it --entrypoint=/bin/bash ubuntu:20.04
WARN[0000] "/" is not a shared mount, this could cause issues or missing mounts with rootless containers
Resolved "ubuntu" as an alias (/etc/containers/registries.conf.d/000-shortnames.conf)
Trying to pull docker.io/library/ubuntu:20.04...
Getting image source signatures
Copying blob edaedc954fb5 done
Copying config 6df8940237 done
Writing manifest to image destination
Storing signatures
root@fd530e95ef25:/# curl https://github.com/fi-ts/cloudctl/releases/download/v0.12.1/cloudctl-linux-amd64 -L -o cloudctl
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 50.4M  100 50.4M    0     0  9496k      0  0:00:05  0:00:05 --:--:-- 10.9M
root@fd530e95ef25:/# md5sum cloudctl
6fc861f0686fa9d82e9198aeafc715e4  cloudctl
root@fd530e95ef25:/# chmod +x cloudctl
root@fd530e95ef25:/# mv cloudctl /usr/local/bin/
root@fd530e95ef25:/# ldd /usr/local/bin/cloudctl
/usr/local/bin/cloudctl: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /usr/local/bin/cloudctl)
/usr/local/bin/cloudctl: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /usr/local/bin/cloudctl)
        linux-vdso.so.1 (0x00007ffe713cc000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f6540a6a000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f6540c61000)
root@fd530e95ef25:/# apt install libc6
Reading package lists... Done
Building dependency tree
Reading state information... Done
libc6 is already the newest version (2.31-0ubuntu9.9).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
root@fd530e95ef25:/# cloudctl version
cloudctl: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by cloudctl)
cloudctl: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by cloudctl)
```

Could you please link the new versions of cloudctl statically again?

**Comment by Gerrit91** [comment]
Thanks for your report, we just merged #253 which brings back the statically linked binary.

**Comment by majst01** [comment]
v0.12.2 should be available in the next few minutes
